iphone/plist.txt

```diff
@@ -53,6 +54,7 @@
     id = Penanda
     it = Segnalibri
     ja = ブックマーク
+    mr = खूणपत्रे
```

Comment:
Please make it in the alphabet order here and in other places.

Reply:
oops. I fixed it now in all places in this file and once again squashed into 2 commits and force pushed.